src/python_pachyderm/client.py

```diff
@@ -61,6 +61,7 @@ def __init__(
         root_certs=None,
         transaction_id=None,
         tls=None,
+        max_message_length=(1024 ** 2) * 20,
```

Comment:
I think it might make sense to make this a constant, since it's a constant in the server. Or do you think people might need to change it?